lib/Dialect/Substrait/IR/Substrait.cpp

```diff
@@ -155,6 +155,22 @@ EmitOp::inferReturnTypes(MLIRContext *context, std::optional<Location> loc,
   return success();
 }
 
+LogicalResult
+FetchOp::inferReturnTypes(MLIRContext *context, std::optional<Location> loc,
+                          ValueRange operands, DictionaryAttr attributes,
+                          OpaqueProperties properties, RegionRange regions,
+                          llvm::SmallVectorImpl<Type> &inferredReturnTypes) {
+  Value input = operands[0];
```

Comment:
Minor nit: it is *maybe* more readable to say `Type inputType = ...` here and remove the `getType` from the next line...?